### ISSUE TITLE
Does Eclipse Che plan to support Eclipse Theia

### ISSUE THREAD

**Issue by OmerMajNition** [eclipse/che#23006]
### Summary

We want to use Theia as an editor running on Eclipse Che, there doesn't seem to have support for deploying Theia on Che. Does Che team plan to support Theia as an editor deployed with Che?

### Relevant information

I see https://github.com/eclipse-che/che/issues/20282 as an open issue

https://github.com/eclipse-theia/theia/discussions/13789 tells further that its upto the Che team to go ahead with Eclipse-Theia support

Is this on the roadmap?
This is a critical piece for us to make decisions on either to go with Eclipse-Theia or not.

**Comment by AObuchow** [comment]
Eclipse Che used to support Eclipse Theia but to my knowledge, it is no longer supported, unfortunately.

**Comment by ibuziuk** [comment]
@OmerMajNition Thank you for your interest in Eclipse Che and Theia. 
Che-Theia was sunset as a built-in editor as part of https://github.com/eclipse-che/che/issues/21771 back in 2022. This was communicated on the Community mailing list as part of  https://www.eclipse.org/lists/che-dev/msg04439.html where more details can be found.

TLDR; There were no volunteers from the Community ready to take over Che-Theia support and, as a result, it was sunset as a built-in Eclipse Che editor.

At this point in time, there are no plans to have Eclipse Theia supported in Eclipse Che unless somebody from the Community is going to take over this work - another thread which describes the PoC might be useful for the context https://www.eclipse.org/lists/che-dev/msg04485.html

**Comment by monaka** [comment]
@AObuchow
FYI: There exists a PoC implementation of Theia on Che by @l0rd . https://github.com/l0rd/theia-editor/

Note that it is not "Che-Theia" but a vanilla version of Theia.

**Comment by ibuziuk** [comment]
Closing, more details in:
https://www.eclipse.org/lists/che-dev/msg04439.html
https://www.eclipse.org/lists/che-dev/msg04485.html
Feel free to open another issue or raise it on the Community Call [1] if somebody from the Community would like to support and invest in Theai integration

[1] https://github.com/eclipse-che/che/wiki/Eclipse-Che-Community-Meetings#next-meeting
